Dockerfile

```diff
@@ -13,14 +29,19 @@ COPY CMakeLists.txt /root/history-tools
 COPY libraries /root/history-tools/libraries
 COPY src /root/history-tools/src
 COPY unittests /root/history-tools/unittests
+
 RUN mkdir /root/history-tools/build
 WORKDIR /root/history-tools/build
-RUN cmake -GNinja -DCMAKE_BUILD_TYPE=Release -DPostgreSQL_TYPE_INCLUDE_DIR=/usr/include/postgresql ..
-RUN ninja && ctest --output-on-failure
+RUN cmake -GNinja -DCMAKE_BUILD_TYPE=Release -DPostgreSQL_TYPE_INCLUDE_DIR=/usr/include/postgresql .. && \
+    ninja && \
+    ctest --output-on-failure
 
 FROM ubuntu:20.04
-RUN apt-get update && apt-get install -y \
-    postgresql-client netcat openssl && \
-    rm -rf /var/cache/apt/lists
+RUN apt-get update && \
+    apt-get install -y \
+        netcat \
+        openssl \
+        postgresql-client && \
+    apt-get clean
 
-COPY --from=builder /root/history-tools/build/fill-pg /usr/local/bin/fill-pg
+COPY --from=builder /root/history-tools/build/fill-pg /usr/local/bin/fill-pg
```

Comment:
POSIX standard is to include a newline (`\n`) at the end of the file.

Reply:
done.